### ISSUE TITLE
What is the WSDL syntax to populate the "Description" column in the html table?

### ISSUE THREAD

**Issue by molund** [chenjianjx/wsdl2html#13]
I'm trying to use the xs:annotation / xs:documentation tags, but nothing is getting populated in the description column of the HTML tables.

```
 <s:element minOccurs="1" maxOccurs="1" name="ContactMethodId" type="s:int" >
      <xs:annotation>
          <xs:documentation>Na = 0, RegularMail = 1, Email = 2, VoiceMail = 3, FaxBack = 4, Print = 5</xs:documentation>
      </xs:annotation>
</s:element>
```

**Comment by chenjianjx** [comment]
You are right.  It's not implemented.  Sorry, don't get bandwidth to do it. Have to parse generated java files and extract javadocs. 

Pull requests are welcome.

**Comment by sco0ter** [comment]
I'd be happy to see this feature as well. It's even trickier for my application: We have Java classes, `org.apache.cxf:cxf-java2ws-plugin` generates WSDL and your plugin generates the documentation from it.

From first glance it's hardly possible to even generate `<xs:documentation>` tags from JavaDoc (with cxf).

**Comment by chenjianjx** [comment]
Just did a research.  This is very hard to implement (Parse generated java files).  And even if you do,  its help is limited: 

- WSDL Documentation for services and operations can be extracted, because they can show up in generated java files . See https://javaee.github.io/metro/doc/user-guide/ch03.html#generating-javadocs-from-wsdl-documentation 
- WSDL Documentation for operations' params can't be extracted because JAX-WS won't put them into java files
- WSDL Documentation in schemas, that is, docs for data classes and their fields can't be extracted because JAX-WS won't put them into java files


Their last one may be the most the important one.   If it can't be done, then the whole efforts are not worth it
